WcaOnRails/app/views/competitions/edit_events.html.erb

```diff
@@ -1,21 +1,18 @@
 <% provide(:title, @competition.name) %>
-<% add_to_packs("edit_events") %>
 
 <%= render layout: 'nav' do %>
   <% if !current_user.can_update_events?(@competition) %>
     <%= alert :warning, note: true do %>
       Events management has been locked after results being posted. Please contact WRT if you want to edit them.
     <% end %>
   <% end %>
-  <div id="events-edit-area"></div>
-  <script>
-    $(function() {
-      window.wca.initializeEventsForm(
-        <%= @competition.id.to_json.html_safe %>,
-        <%= current_user.can_add_and_remove_events?(@competition).to_json.html_safe %>,
-        <%= current_user.can_update_events?(@competition).to_json.html_safe %>,
-        <%= @competition.events_wcif.to_json.html_safe %>
-      );
-    });
-  </script>
+  <%= react_component("EditEvents/index", {
+    competitionId: @competition.id,
+    canAddAndRemoveEvents: current_user.can_add_and_remove_events?(@competition),
+    canUpdateEvents: current_user.can_update_events?(@competition),
+    canUseQualifications: current_user.can_update_qualifications?(@competition),
```

Comment:
Because I created it in JS world without knowing the Ruby code existed, then later went to hook it up to the existing code (I think). I'll change it to match.